src/components/navBar/navBar.js

```diff
@@ -252,20 +270,28 @@ MdNavBarController.prototype._updateTabs = function(newValue, oldValue) {
  * Repositions the ink bar to the selected tab.
  * @private
  */
-MdNavBarController.prototype._updateInkBarStyles = function(tab, newIndex, oldIndex) {
-  this._inkbar.toggleClass('_md-left', newIndex < oldIndex)
-      .toggleClass('_md-right', newIndex > oldIndex);
```

Comment:
I can run this through presubmits once the other code review changes are made, but the removal of these classes here and in the SCSS (even if marked private) may caught some apps to break. If that is the case and the presubmits fail, this will have to be targeted at 1.2.0.